plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergStatistics.java

```diff
@@ -177,8 +196,8 @@ public void registerDataFile(DataFile dataFile, PartitionSpec partitionSpec)
                     }
                 }
                 else {
-                    Object lowerBound = lowerBounds == null ? null : convertIcebergValueToTrino(column.type(), lowerBounds.get(id));
-                    Object upperBound = upperBounds == null ? null : convertIcebergValueToTrino(column.type(), upperBounds.get(id));
+                    Object lowerBound = lowerBounds.map(bounds -> convertIcebergValueToTrino(column.type(), bounds.get(id))).orElse(null);
+                    Object upperBound = upperBounds.map(bounds -> convertIcebergValueToTrino(column.type(), bounds.get(id))).orElse(null);
```

Comment:
lowerBounds, upperBounds are unnecessarily Optional. "No entry" (null) is treated the same as "no map at all".